data/crac-creation/crac-creator-cim/src/main/java/com/powsybl/openrao/data/craccreation/creator/cim/craccreator/CimCracUtils.java

```diff
@@ -40,4 +52,34 @@ public static Set<FlowCnec> getFlowCnecsFromCrac(MonitoredSeries monitoredSeries
         return mscc.getCreatedCnecIds().stream().map(cnecId -> cracCreationContext.getCrac().getFlowCnec(cnecId)).collect(Collectors.toSet());
     }
 
+    public static String getCurveTypeFromTimeSeries(TimeSeries cimTimeSerie) {
+        String curveType = cimTimeSerie.getCurveType();
+        if (StringUtils.isBlank(curveType)) {
+            curveType = DEFAULT_CURVE_TYPE;
+        }
+        if (!curveType.equals("A03") && !curveType.equals("A01")) {
```

Comment:
to explicit what these mean, define in CimConstants:
SEQUENTIAL_FIXED_SIZE_BLOCKS_CURVE_TYPE = "A01"
VARIABLE_SIZED_BLOCK_CURVE = "A03"